language/compiler/ir_to_bytecode/syntax/src/lexer.rs

```diff
@@ -294,7 +294,7 @@ pub struct __Matcher<'input, 'builder> {
 }
 
 impl<'input, 'builder> __Matcher<'input, 'builder> {
-    pub fn next(&mut self) -> Result<(), ParseError<usize, Token<'input>, failure::Error>> {
+    pub fn advance(&mut self) -> Result<(), ParseError<usize, Token<'input>, failure::Error>> {
         let __text = self.text.trim_start();
```

Comment:
Out of curiosity, what's with the funny named variables like `__text`?

Reply:
This lexer code was partly copied from the output of lalrpop with the old parser. It needs to be completely ripped out and replaced, so I didn't bother cleaning it up very much. Apparently lalrpop uses a convention of prefixing its own symbols with underscores to avoid name conflicts.